app/reducers/icoParameters.ts

```diff
@@ -0,0 +1,83 @@
+import * as moment from "moment";
+import { IcoPhase, LOAD_ICO_PARAMS, NEW_PHASE_ACTION } from "../actions/constants";
+import { envPayload } from "../config";
+
+export interface IcoParametersState {
+  loading: boolean;
+  address: string;
+  startDate: string;
+  endDate: string;
+  minCap: number;
+  maxCap: number;
+  icoPhase: IcoPhase;
+}
+
+interface IStandardReduxAction<T> {
```

Comment:
this should be in file that makes it easy to reuse it. Maybe file named `types`?